src/platform/libretro/libretro.c

```diff
@@ -422,9 +516,14 @@ void static _setupMaps(struct mCore* core) {
 }
 
 void retro_reset(void) {
-	core->reset(core);
-	_setupMaps(core);
-
+	core1->reset(core1);
+	_setupMaps(core1);
+	core2->reset(core2);
+	_setupMaps(core2);
```

Comment:
I'm about 80% sure calling this function four times will leak memory or otherwise misbehave. You're hardcoding core1 anyways, remove the other three calls. The resets are fine.

Reply:
You mean the setupMaps stuff?
I just did everything 4x but I wasn't sure of that yeah